configs/sites/tier1/hera/mirrors.yaml

```diff
@@ -1,15 +1,15 @@
 mirrors:
   local-source:
     fetch:
-      url: file:///contrib/spack-stack/mirror
+      url: file:///scratch1/NCEPDEV/nems/role.epic/spack-stack/source-cache
```

Comment:
please revert

Reply:
I moved (large) mirror directory as suggested by @AlexanderRichert-NOAA , we are not supposed to store large datasets on /contrib space.

Reply:
The /contrib space is needed for performance/stability of the installed packages themselves so that model developers aren't held up by parallel filesystem issues, but we don't need that for the source/build caches (i.e., we're not in "real" trouble if the filesystem crashes for a day or two and we can't access the mirrors). The admins have already been fairly generous as far as disk usage/contrib policies/etc., so I think it would be best to keep the caches in "bulk storage" on the parallel filesystem. That said, I would suggest making the same change for Jet unless there's some reason not to.

Reply:
I thought I did it for Jet as well, now I see I didn't. I'll do it over there as well.

Reply:
Thanks, that makes sense.